pkg/async/client/asynq.go

```diff
@@ -34,25 +39,21 @@ type (
 	}
 )
 
-func New(log *slog.Logger, redis *redis.Client) *Client {
+func New(log *slog.Logger, redis *redis.Client, opts ...asynq.Option) *Client {
 	client := asynq.NewClientFromRedisClient(redis)
 
-	opts := []asynq.Option{
-		asynq.MaxRetry(5),
-		asynq.Timeout(20 * time.Minute),
+	// Set default opts
+	if len(opts) == 0 {
+		opts = append(opts, defaultAsynqRetries, defaultAsynqTimeout)
 	}
```

Comment:
Unsure about if there are conflicts, but what about always prepending the default options?

Like in `append([]asynq.Option{defaultAsynqRetries, defaultAsynqTimeout}, opts...)`?
Shouldn't `opts` then override the defaults? Would be less confusing when adding one additional option for users of the asynq client.

Reply:
done